mso/resource_mso_schema_site_vrf_region_cidr_subnet.go

```diff
@@ -106,7 +118,7 @@ func resourceMSOSchemaSiteVrfRegionCidrSubnetImport(d *schema.ResourceData, m in
 	stateCidr := import_split[1]
 	stateIp := import_split[2]
 
-	for i := 0; i < count; i++ {
+	for i := 0; i < count && !found; i++ {
```

Comment:
It would be good to include the template name as well in the import statement.

terraform import mso_schema_site_vrf_region_cidr_subnet.sub1 {schema_id}/site/{site_id}-{template_name}/vrf/{vrf_name}/region/{region_name}/cidrIP/{cidr_ip}/subnet/{ip}

After that we can retrieve the name of the template from the import statement through string manipulation and use it as part of the condition -> if apiSite == stateSite && apiTemplate == stateTemplate { (same as Read)

Reply:
same as above